controllers/issue.js

```diff
@@ -260,8 +260,8 @@ exports.view = function (aReq, aRes, aNext) {
       }
 
       // Discussion
-      discussion = options.discussion = modelParser.parseDiscussion(aDiscussion);
-      modelParser.parseIssue(discussion);
+      options.discussion = discussion = modelParser.parseDiscussion(aDiscussion);
+      modelParser.parseIssue(discussion); // WARNING: Return value never used
```

Comment:
@sizzlemctwizzle Cc: @Zren 
What's this about? Doing a model parse and not returning it's values into the object?